ra/ra_test.go

```diff
@@ -1252,6 +1252,23 @@ func TestCheckFQDNSetRateLimitOverride(t *testing.T) {
 	test.AssertNotError(t, err, "FQDN set certificate per name exemption not applied correctly")
 }
 
+func TestDeactivateAuthorization(t *testing.T) {
+	_, sa, ra, _, cleanUp := initAuthorities(t)
+	defer cleanUp()
+	authz := core.Authorization{RegistrationID: 1}
+	authz, err := sa.NewPendingAuthorization(ctx, authz)
+	test.AssertNotError(t, err, "Could not store test data")
+	authz.Status = core.StatusValid
+	err = sa.FinalizeAuthorization(ctx, authz)
+	test.AssertNotError(t, err, "Could not store test data")
+	fmt.Println(authz)
```

Comment:
Should this `Println` have been dropped? Its output will get squashed by `go test` without a `-v` but it might be better to remove outright in this case.